ocs_ci/ocs/machine.py

```diff
@@ -262,6 +262,24 @@ def add_node(machine_set, count):
     return True
 
 
+def delete_node(machine_set, count):
+    """
+    Delete nodes from the cluster by decreasing the number of replicas in the
+    machine set.
+    Args:
+        machine_set (str): Name of a machine set to get decrease replica count
+        count (int): Count to decrease by
+    Returns:
+        bool: True if commands executes successfully
+    """
+    replica_count = get_replica_count(machine_set) - count
+    ocp = OCP(namespace=constants.OPENSHIFT_MACHINE_API_NAMESPACE)
+    ocp.exec_oc_cmd(
+        f'scale --replicas={replica_count} machinesets {machine_set}'
+    )
+    return True
```

Comment:
If provided machine_set  has more then 1 replica count it might happen that it will delete ocs node and cluster will go into degraded state

Reply:
this will decrease the number of machines in the machineses by delete the non-OCS machines.

Reply:
I think Pratik's point is true that there is a chance that one of the OCS nodes is removed. Maybe, you should create a new machine set to avoid this issue.

Reply:
Please check - please check - https://docs.openshift.com/container-platform/4.1/machine_management/creating-machineset.html for creating new machine set.